caveclient/auth.py

```diff
@@ -206,6 +209,21 @@ def save_token(
             self._token_key = token_key
             self._token_file = save_token_file
 
+    def get_users(self, user_id):
```

Comment:
better function name: `get_user_information`. Please use plural (`user_ids`) or add functionality to handle a single ID